readthedocs/core/fields.py

```diff
@@ -0,0 +1,11 @@
+"""Shared model fields and defaults"""
+
+import sys
+from random import randrange
+
+from django.utils.http import int_to_base36
+
+
+def default_token():
+    """Generate default value for token field"""
+    return ''.join([int_to_base36(randrange(sys.maxint)) for _ in range(2)])
```

Comment:
Does this need to be cryptographically secure, or is it just a hash?  I feel like Django has a way to generate these that can't be guessed.

Either way, we should document the thread profile of it.